parsl/configs/improv.py

```diff
@@ -0,0 +1,33 @@
+from parsl.config import Config
+from parsl.executors import HighThroughputExecutor
+from parsl.launchers import MpiRunLauncher
+from parsl.providers import PBSProProvider
+
+config = Config(
+        executors=[
+            HighThroughputExecutor(
+                label="Improv_multinode",
+                max_workers_per_node=32,
+                provider=PBSProProvider(
+                    account="YOUR_ALLOCATION_ON_IMPROV",
+                    # PBS directives (header lines): for array jobs pass '-J' option
+                    # scheduler_options='#PBS -J 1-10',
+                    scheduler_options='',
+
+                    queue="compute",
+                    # Command to be run before starting a worker, such as:
+                    # 'module load Anaconda; source activate parsl_env'.
+                    # **WARNING** Improv requires an openmpi module to be
```

Comment:
maybe the example here should include loading an openmpi module, either in the explicit value or in the comment on this line

Reply:
oh you did, i misread `'''` as `''`